ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm

```diff
@@ -0,0 +1,17 @@
+Value LOCAL_PORT (\S+)
+Value REMOTE_DEVICE (\S+)
+Value REMOTE_PORT_ID (\S+)
+Value CAPABILITY (R|B|O|RO|RB|BR|BO|OR|OB|RBO|ROB|BRO|BOR|ORB|OBR)
+Value REMOTE_PORT_DESC (\S+)
+
+Start
+  ^Capability\s+codes:\s+\(R\)\s+Router,\s+\(B\)\s+Bridge,\s+\(O\)\s+Other
+  ^LocalPort\s+RemoteDevice\s+RemotePortID\s+Capability\s+RemotePortDescr
+  ^[-+\s+]+ -> Row
+  ^. -> Error
+
+Row  
+  ^${LOCAL_PORT}\s+${REMOTE_DEVICE}\s+${REMOTE_PORT_ID}\s+${CAPABILITY}\s+${REMOTE_PORT_DESC} -> Record
```

Comment:
```suggestion
  ^${LOCAL_INTERFACE}\s+${NEIGHBOR}\s+${NEIGHBOR_INTERFACE}\s+${CAPABILITY}\s+${REMOTE_PORT_DESC} -> Record
```

Reply:
fixed in [ef74743](https://github.com/networktocode/ntc-templates/pull/923/commits/ef74743e064a2984fd28a620dddc342f8ab1534f)